packages/devtools_app/lib/src/service_registrations.dart

```diff
@@ -61,3 +61,7 @@ const flutterMemory = RegisteredServiceDescription._(
 const flutterListViews = '_flutter.listViews';
 
 const displayRefreshRate = '_flutter.getDisplayRefreshRate';
+
+/// Flutter engine returns estimate how much memory is used by layer/picture raster
+/// cache entries in bytes.
+const flutterEngineRasterCache = '_flutter.estimateRasterCacheMemory';
```

Comment:
nit: maybe keep the word estimate in the name to make it clear that this data may be a bit innacurate.

Reply:
done